### DISCUSSION TITLE
can't upgrade go mod from 224 to 300

### DISCUSSION THREAD

**Discussion by bighunter513** [ergo-services/ergo#223] [Q&A]
**Describe the bug**
when use go-mod-upgrade to upgrade ergo:

Updating github.com/ergo-services/ergo to version 1.999.300...
   ⨯ Error while updating module error=exit status 1 
name=github.com/ergo-services/ergo 
out=go: github.com/ergo-services/ergo@upgrade (v1.999.300) 
requires github.com/ergo-services/ergo@v1.999.300: 
parsing go.mod:
        module declares its path as: ergo.services/ergo
                but was required as: github.com/ergo-services/ergo

**Comment by halturin** [comment]
starting from 3.0 this module lives on its own domain `ergo.services`. its been made to get rid of GitHub-lock. 

Besides that, the new version has an entirely new design and is not compatible with the previous one
